modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1302,11 +1302,8 @@
     <string name="settings_privacy_crash_link">Link your account to crashes</string>
     <string name="settings_privacy_crash_link_summary">Allow us to link your Pocket Casts account to crash reports.</string>
 
-    <string name="settings_advanced_no_options">Enable background refresh to see advanced options</string>
     <string name="settings_advanced_sync_on_metered">Sync on Metered Networks</string>
-    <string name="settings_advanced_sync_on_metered_on">While on a metered network, synchronization will occur automatically in the background and when the application is opened.</string>
-    <string name="settings_advanced_sync_on_metered_off">While on a metered network, synchronization will only occur if performed manually.</string>
-
+    <string name="settings_advanced_sync_on_metered_summary">When enabled (default), synchronization will occur automatically in the background and when the application is opened.\n\nWhen disabled, synchronization will only occur if you manually refresh. Disable this setting if you prefer to avoid automatic playlist modifications and data usage while on mobile data.\n\nThis setting is only applicable when background refresh is turned on.</string>
```

Comment:
It would be good to shorten this up. What do you think about this?

> Permits automatic background synchronization on metered networks. This is enabled by default and has no effect if background refresh is turned off.